test/qa/test_connect.py

```diff
@@ -491,20 +492,21 @@ def test_status_connected(tech, proto, obfuscated):
         name, hostname = server.get_hostname_by(technology=tech, protocol=proto, obfuscated=obfuscated)
         sh.nordvpn.connect(hostname.split(".")[0])
 
-        connect_time = int(time.time())
+        connect_time = time.monotonic()
 
         data_10MB = requests.get("http://speedtest.tele2.net/10MB.zip")
-        destination_file = "file_10MB"
-        if data_10MB.status_code == 200:
-            with open(destination_file, "wb") as file:
-                file.write(data_10MB.content)
+        data_10MB.raise_for_status()
+        assert data_10MB.status_code == 200
```

Comment:
Don't need to assert if using `raise_for_status` already, the exception from that function will fail the test anyway

Reply:
Right, removed assert 👍